Client/Frontend/Browser/URLBarView.swift

```diff
@@ -406,7 +406,9 @@ class URLBarView: UIView {
                 }
             })
         } else {
-            self.progressBar.alpha = 1.0
+            if self.progressBar.alpha < 1.0 {
```

Comment:
Nit: Do we need to check if it's less than 1.0? I think we can just always assign it 1.0 because if it's not 1.0, we're going to set it to 1 anyways